src/highrl/envs/robot_env.py

```diff
@@ -73,7 +74,9 @@ def __init__(
         self.robot_goal_py: int = 0
 
         self.is_initial_state = True
-
+        self.tensorboard_dir = "runs"
```

Comment:
To separate the tensorboard logs for the teacher and robot modules, rewrite this as
```python
self.tensorboard_dir = "runs/robot"
```